guides/mapping-and-blending/ontology-mapping/README.md

```diff
@@ -1,3 +1,6 @@
+
+# Ontology mapping is a legacy concept/code. Please use [Blending](../blending/README.md).
+
```

Comment:
Since ontology mapping shouldn't be used at all in 2.4, I suggest deleting the content on the page and write something like this instead: 
```
Ontology mapping has been replaced by [Blending](../blending/README.md) in Starcounter 2.4. Use the dropdown on the left side to read about how it worked in earlier versions.
```

Same applies to the URI mapping page.